Lib/test/support/__init__.py

```diff
@@ -1168,6 +1168,16 @@ def requires_limited_api(test):
         return unittest.skip('needs _testcapi and _testlimitedcapi modules')(test)
     return test
 
+
+TEST_MODULES_ENABLED = sysconfig.get_config_var('TEST_MODULES') == 'yes'
+
+
+def requires_test_modules(test):
+    if TEST_MODULES_ENABLED != 'yes':
```

Comment:
Currently it is a string. Perhaps a bit unfortunate, since the name suggests a boolean. We should change it to a boolean IMO.

As you point out, the demand is not huge. I'm fine with either way.

Reply:
But it is defined as a boolean.

Reply:
Maybe it is; I'll check when I get to my computer. Anyway: if it is a boolean, let's keep it that way; if it's a string, let's make it a boolean ;)